[2024-01-18T22:09:14.409+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_theta scheduled__2024-01-01T03:30:00+00:00 [queued]>
[2024-01-18T22:09:14.451+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_theta scheduled__2024-01-01T03:30:00+00:00 [queued]>
[2024-01-18T22:09:14.452+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:09:14.452+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T22:09:14.453+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:09:14.501+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): train_model_theta> on 2024-01-01 03:30:00+00:00
[2024-01-18T22:09:14.513+0300] {standard_task_runner.py:55} INFO - Started process 6025 to run task
[2024-01-18T22:09:14.541+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'train_model_theta', 'scheduled__2024-01-01T03:30:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmp0aq3qutk']
[2024-01-18T22:09:14.547+0300] {standard_task_runner.py:83} INFO - Job 45: Subtask train_model_theta
[2024-01-18T22:09:14.692+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.train_model_theta scheduled__2024-01-01T03:30:00+00:00 [running]> on host xflow
[2024-01-18T22:09:14.884+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=train_model_theta
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T03:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T03:30:00+00:00
[2024-01-18T22:09:14.889+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T22:09:14.891+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/train_model.py theta']
[2024-01-18T22:09:14.935+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T22:09:32.204+0300] {subprocess.py:93} INFO - Registered model 'theta_model' already exists. Creating a new version of this model...
[2024-01-18T22:09:32.274+0300] {subprocess.py:93} INFO - Created version '6' of model 'theta_model'.
[2024-01-18T22:09:33.338+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T22:09:33.387+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=train_model_theta, execution_date=20240101T033000, start_date=20240118T190914, end_date=20240118T190933
[2024-01-18T22:09:33.442+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T22:09:33.487+0300] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
